tests/unit/test_get_movie_by_title.py

```diff
@@ -1 +1,9 @@
 # TODO: Feature 3
+from src.models.movie import Movie
+from app import movie_repository
+from src.repositories.movie_repository import get_movie_repository
+
+def get_movie_test():
+    movie_repository.create_movie("title1","director1",3)
+    movie = movie_repository.get_movie_by_title("title1")
+    assert movie
```

Comment:
I don't believe this is a valid test, as it isn't comparing anything.

Reply:
I was intending to check if the movie object was empty or not, and forgot the '!= "" ' part. I got it all fixed up properly now.